test/cypress/integration/settings/your-psp.cy.js

```diff
@@ -348,6 +354,95 @@ describe('Your PSP settings page', () => {
     })
   })
 
+  describe('When using a Worldpay account with recurring payments enabled', () => {
+    it('should render the page correctly when there are no existing credentials', () => {
+      cy.task('setupStubs', getUserAndGatewayAccountStubs({
+        gateway: 'worldpay',
+        requires3ds: true,
+        recurringEnabled: true,
+        integrationVersion3ds: 1,
+        gatewayAccountCredentials: [{
+          payment_provider: 'worldpay',
+          credentials: {},
+          external_id: credentialExternalId,
+          id: credentialsId
+        }],
+        validateCredentials: testCredentialsMOTO
```

Comment:
This probably isn't needed but happy for it to be changed in a follow up

Reply:
Thanks - good spot